arangod/VocBase/voc-types.h

```diff
@@ -23,16 +23,7 @@
 
 #pragma once
 
-#include <cstdint>
```

Comment:
Including `cstdint` seems to make sense here, at least a few lines below we are referring to `uint64_t`. I think we should keep this include here.

Reply:
Well, `cstdint` defines `std::unint64_t`. Since we use `uint64_t`,  we should actually include `stdint.h`.

Reply:
Yeah, it's awful. But c-headers, ".h" was depricated, so imo better to use std::size_t/uint/int/etc